app/queries/offers_to_chase_query.rb

```diff
@@ -4,9 +4,12 @@
 class OffersToChaseQuery
   def self.call(chaser_type:, date_range:)
     ApplicationChoice
+      # Filter out candidates who should not receive emails about their accounts
+      .joins(:candidate).merge(Candidate.for_marketing_or_nudge_emails)
```

Comment:
Very minor and me being fussy but could we have the `.merge` on the line below to aid readability and to follow the existing format?

Reply:
Sure